test/context.test.js

```diff
@@ -25,6 +25,15 @@ describe('Context', function () {
   })
 
   describe('repo', function () {
+    it('returns error for context.repo()', function () {
+      context.payload.repository = undefined
+      try {
+        context.repo()
+      } catch (e) {
+        expect(String(e)).toEqual('Error: Sorry not available for this event')
```

Comment:
This'd be a good place for Jest's [snapshot testing](https://facebook.github.io/jest/docs/en/snapshot-testing.html), so that we aren't hard-coding an expected outcome. It makes it easier for us to change in the future.

```js
expect(e.message).toMatchSnapshot()
```